.github/workflows/android_alarm_manager_plus.yaml

```diff
@@ -40,26 +41,33 @@ jobs:
   android_integration_test:
     runs-on: macos-latest
     timeout-minutes: 30
+    strategy:
+      matrix:
+        android-api-level: [21, 26, 32]
+
     steps:
       - name: "Checkout repository"
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: "Install Flutter"
         run: ./.github/workflows/scripts/install-flutter.sh stable
       - name: "Install Tools"
         run: ./.github/workflows/scripts/install-tools.sh
 
-      # Required for Android builds
-      - uses: actions/setup-java@v1
+        # Required for Android builds
+      - uses: actions/setup-java@v2
         with:
+          distribution: "temurin"
           java-version: "11"
 
       - name: "Bootstrap Workspace"
         run: melos bootstrap
       - name: "Android Integration Test"
         uses: reactivecircus/android-emulator-runner@v2
         with:
-          api-level: 30
-          arch: x86_64
+          api-level: ${{ matrix.android-api-level }}
+          cores: 3
```

Comment:
`MacOS` runner [has 3 cores available](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources), while `android-emulator-runner` uses 2 by default. Specifying this value in effort to get higher performance.